backend/docs/internal_v1_deployments.oai.yaml

```diff
@@ -0,0 +1,1130 @@
+openapi: 3.0.1
+info:
+  description: |
+    Internal API of deployments service
+  title: Deployments Internal API
+  version: "1"
+servers:
+  - url: http://mender-deployments:8080/api/internal/v1/deployments
+paths:
+  /health:
+    get:
+      operationId: Check Health
+      responses:
+        "204":
+          content: {}
+          description: |
+            Service is healthy and all dependencies are up and running.
+        "500":
+          content:
+            application/json:
+              example:
+                error: internal error
+                request_id: ffd712be-d697-4cb7-814b-88ff1e2eb5f6
+              schema:
+                $ref: "#/components/schemas/Error"
+          description: |
+            Unexpected internal error
+        "503":
+          content:
+            application/json:
+              example:
+                error: "error reaching MongoDB: context deadline exceeded"
+                request_id: ffd712be-d697-4cb7-814b-88ff1e2eb5f6
+              schema:
+                $ref: "#/components/schemas/Error"
+          description: |
+            Service unhealthy / not ready to accept traffic. At least one dependency is not running.
+      summary: Check the health of the service
+      tags:
+        - Internal API
+  /alive:
+    get:
+      operationId: Check Liveliness
+      responses:
+        "204":
+          content: {}
+          description: Service is up and running.
+        "500":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+          description: |
+            Internal API error
+      summary: |
+        Trivial endpoint that unconditionally returns an empty 200 response whenever the API handler is running correctly.
```

Comment:
I know how it is and was, but either this is not true or the responses are not matching.